lib/chrome-driver.js

```diff
@@ -24,7 +23,7 @@ ChromeDriver.prototype.start = function () {
     cwd: process.cwd(),
     env: this.getEnvironment()
   }
-  this.process = ChildProcess.spawn(process.execPath, args, options)
+  this.process = ChildProcess.spawn(this.path, args, options)
```

Comment:
This ends up getting to here: https://github.com/kevinsawicki/spectron/blob/93aa264421252bf229ca91b3d63a0bd1475a6b41/lib/application.js#L127

and must hang inside the .init() call because it never resolves or rejects (I have a timeout set for 2 minutes).